Index: coreutils-8.26/Makefile.in
===================================================================
--- coreutils-8.26.orig/Makefile.in	2016-11-30 13:34:55.000000000 -0500
+++ coreutils-8.26/Makefile.in	2017-02-22 07:18:55.352394058 -0500
@@ -14661,6 +14661,10 @@
 
 $(TEST_LOGS): $(PROGRAMS)
 
+ifeq ($(DEB_TARGET_ARCH), ppc64el)
+lib/sha256.o: CFLAGS+=-fno-schedule-insns
+endif
+
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
 .NOEXPORT:
